CompressImagesFunction/ImageQuery.cs

```diff
@@ -10,8 +11,8 @@ public static class ImageQuery
         public static string[] FindImages(string localPath, RepoConfiguration repoConfiguration)
         {
             var images = KnownImgPatterns.ImgPatterns
-                .AsParallel()
-                .SelectMany(pattern => Directory.EnumerateFiles(localPath, pattern, SearchOption.AllDirectories));
+                .SelectMany(pattern => Directory.EnumerateFiles(localPath, "*.*", SearchOption.AllDirectories)
+                .Where(fn => pattern.Contains(Path.GetExtension(fn).ToLower())));
```

Comment:
I encountered a bug with this. 

Since we are querying all files now, we get files that have no extension.

`Path.GetExtension("path/to/randomfile")` is `""`

then `"*.png".Contains("")` is `true`

I have a fix in progress I'll open a PR soon